### PR TITLE
Feature/api update

### PR DESCRIPTION
**Pull request by druiz0992**

### CODE REVIEW

**Review by davidsrz — DISMISSED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-> Hermez is currently under development. Some of the details in the documentation can be modified before mainnet network launch.
-
 Welcome to the [Hermez](https://hermez.io) documentation.
 
 Hermez is a ZK-Rollup, powered by iden3, focused on scaling payments and token transfers on top of Ethereum.
-Check out [Hermez Testnet](users/testnet.md) now!!
+Check [Hermez](users/mainnet.md) out now!!
+
+Hermez is also available in Rinkeby testnet [here](users/testnet.md) now!!
```
Comment (krlosMata) [inline docs/README.md:8]:
```suggestion
Hermez is also available in Rinkeby testnet [here](users/testnet.md) now !!
```

File: docs/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Hermez network docs
 
-> Hermez is currently under development. Some of the details in the documentation can be modified before mainnet network launch.
-
 Welcome to the [Hermez](https://hermez.io) documentation.
 
 Hermez is a ZK-Rollup, powered by iden3, focused on scaling payments and token transfers on top of Ethereum.
-Check out [Hermez Testnet](users/testnet.md) now!!
+Check [Hermez](users/mainnet.md) out now!!
```
Comment (krlosMata) [inline docs/README.md:6]:
```suggestion
Check [Hermez](users/mainnet.md) out now !!
```

File: docs/developers/api.md [hunk 3]
```diff
@@ -10,3 +10,7 @@ Example of these apps include:
 
 The documentation of the API can be found [here](https://apidoc.hermez.network/)
 
+Hermez Testnet API can be found [here](https://api.testnet.hermez.io/v1)
```
Comment (krlosMata) [inline docs/developers/api.md]: - I would add a link that returns the state of the network
- I would set API Mainnet before Testnet API 

```suggestion
Hermez Testnet API can be found [here](https://api.testnet.hermez.io/v1/state)
```

File: docs/developers/api.md [hunk 4]
```diff
@@ -10,3 +10,7 @@ Example of these apps include:
...
+
+Hermez Mainnet API can be found [here](https://api.hermez.io/v1)
```
Comment (krlosMata) [inline docs/developers/api.md]:
```suggestion
Hermez Mainnet API can be found [here](https://api.hermez.io/v1/state)
```

File: docs/users/mainnet.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Hermez
```
Comment (krlosMata) [inline docs/users/mainnet.md]:
```suggestion
# Mainnet
```

File: docs/users/mainnet.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+Hermez is now live on Ethereum. 
+
+## Tools
+- [Web wallet](https://wallet.hermez.io/login) to interact with Hermez ZK-Rollup user accounts
+- Hermez L2 [batch explorer](https://explorer.hermez.io/)
+- Hermez [SDK](https://github.com/hermeznetwork/hermezjs)
+- Hermez [API](https://api.hermez.io/v1)
```
Comment (krlosMata) [inline docs/users/mainnet.md]:
```suggestion
- Hermez [API](https://api.hermez.io/v1/state)
```

File: docs/users/mainnet.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+## Useful Links
+- To interact with the Testnet, you'll need a [Metamask](https://metamask.io/) wallet.
```
Comment (krlosMata) [inline docs/users/mainnet.md]:
```suggestion
- To interact with hermez, you'll need a [Metamask](https://metamask.io/) wallet.
```

File: docs/users/mainnet.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+## Contract Addresses
+Hermez uses 4 main smart contracts:
+- HermezAuctionProtocol: [0x15468b45ed46c8383f5c0b1b6cf2ecf403c2aec2](https://etherscan.io/address/0x15468b45ed46c8383f5c0b1b6cf2ecf403c2aec2)
+- HermezAddress: [0xa68d85df56e733a06443306a095646317b5fa633](https://etherscan.io/address/0xa68d85df56e733a06443306a095646317b5fa633)
+- HermezWithdrawalDelayeAddress: [0x392361427ef5e17b69cfdd1294f31ab555c86124](https://etherscan.io/address/0x392361427ef5e17b69cfdd1294f31ab555c86124)
```
Comment (krlosMata) [inline docs/users/mainnet.md]:
```suggestion
- HermezWithdrawalDelayerAddress: [0x392361427ef5e17b69cfdd1294f31ab555c86124](https://etherscan.io/address/0x392361427ef5e17b69cfdd1294f31ab555c86124)
```

File: docs/users/testnet.md [hunk 9]
```diff
@@ -19,4 +19,4 @@ Hermez uses 4 main smart contracts:
 - HermezWithdrawalDelayeAddress: [0xeFD96CFBaF1B0Dd24d3882B0D6b8D95F85634724](https://rinkeby.etherscan.io/address/0xeFD96CFBaF1B0Dd24d3882B0D6b8D95F85634724)
```
Comment (krlosMata) [inline docs/users/testnet.md]:
```suggestion
- HermezWithdrawalDelayerAddress: [0xeFD96CFBaF1B0Dd24d3882B0D6b8D95F85634724](https://rinkeby.etherscan.io/address/0xeFD96CFBaF1B0Dd24d3882B0D6b8D95F85634724)
```

File: docs/users/testnet.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ Hermez testnet is now live on the Ethereum Rinkeby network. Checkout our [blogpo
 - [Web wallet](https://wallet.testnet.hermez.io/login) to interact with Hermez ZK-Rollup user accounts
 - Hermez L2 [batch explorer](https://explorer.testnet.hermez.io/)
 - Hermez [SDK](https://github.com/hermeznetwork/hermezjs)
-- Hermez [API](https://api.testnet.hermez.io/)
+- Hermez [API](https://api.testnet.hermez.io/v1)
```
Comment (krlosMata) [inline docs/users/testnet.md]:
```suggestion
- Hermez [API](https://api.testnet.hermez.io/v1/state)
```

**Review by krlosMata — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (druiz0992) [reply docs/README.md:8]: There is no space before exclamation mark

**Pull request merged**